### DISCUSSION TITLE
minimal regolith system with ubuntu-server: no instruction for 24.04

### DISCUSSION THREAD

**Discussion by wighawag** [regolith-linux/regolith-desktop#1050] [General]
wanted to install a minimal regolith system with ubuntu-server and I noticed that the instruction are not yet there for 24.04 : https://regolith-desktop.com/docs/howtos/minimal-ubuntu-install/

Is it not possible yet or is the doc just not up to date ?

**Comment by w18g** [comment]
The docs for that page is not yet up to date. But on the main install page, this will work for you. 
https://regolith-desktop.com/docs/using-regolith/install/

**Reply by clarkevans** [reply]
I installed using the Rgolith 3.2 instructions on a fresh Ubuntu 24.04 system with X11 (since I have NVidia, this is what Ubuntu 24.04 defaulted to). This worked, except I some how ended up with duplicate key bindings.

What is unobvious... switching the WM is a small icon in the lower-right corner.
